### PR TITLE
[DotEnv][WebLink][Templating][ErrorHandler] Updated README with minimal example

### PR DESCRIPTION
**Pull request by wouterj**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/13706

We're about to delete some more component docs. The minimal examples where copy/pasted from the original component docs.

### CODE REVIEW

**Review by yceruto — APPROVED** [review]

**Comment by fabpot** [comment]
Why is the `[Documentation]` link not removed for all these components?

**Comment by wouterj** [comment]
> Why is the [Documentation] link not removed for all these components?

There is no documentation link in the ErrorHandler README already. For the others, I think it makes sense to link to the main guide if there is one available. I discovered I forgot the Templating doc link btw, so that's fixed now.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @wouterj.

File: src/Symfony/Component/Dotenv/README.md [hunk 1]
```diff
@@ -4,10 +4,32 @@ Dotenv Component
...
+$dotenv->overload(__DIR__.'/.env');
+
+// loads .env, .env.local, and .env.$APP_ENV.local or .env.$APP_ENV
+$dotenv->loadEnv(__DIR__.'/.env');
+```
+
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/components/dotenv.html)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Dotenv/README.md:10]: for my understanding, are we really going to delete this doc page?

Comment (wouterj) [reply src/Symfony/Component/Dotenv/README.md:10]: Yes, I think so. The API of this component is quite simple (only 3 methods of the same class where documented - which are now shown in the example). In symfony/symfony-docs#13706, I've moved the `.env` syntax to the main configuration guide. Finally, the `loadEnv` logic is explained in both the main guide and the PHPdocs.

So I think this is a component where no knowledge is lost when removing its component docs.

**Pull request merged**